core/Model.php

```diff
@@ -44,32 +44,35 @@ public function __construct()
     }
 
     protected function fetch($sql, $params = [], $returnResultSet = 'result') {
+        $result = false;
```

Comment:
Does this change actually do anything?  No matter what you're still echoing out an Exception message.  at this point the site is broken anyway so is there a benefit to returning false that I'm not seeing?